tests/client/actions/companiesGetCompany.test.js

```diff
@@ -1,13 +1,16 @@
 import {
-  getCompany,
-  RECEIVE_COMPANY
+  getCompanyInfo,
+  RECEIVE_COMPANY_INFO
 } from '../../../client/actions/companies'
 import {SHOW_ERROR} from '../../../client/actions'
 
 jest.mock('superagent', () => {
   return {
     get: (url) => {
-      if (url.split('/')[1] === 'api') {
+      const splitUrl = url.split('/')
+      const id1 = Number(splitUrl[splitUrl.length - 1])
+      const id2 = Number(splitUrl[splitUrl.length - 1])
+      if (id1 === id2) {
```

Comment:
Please add a comment here to explain why we're doing this. Also, mark it TODO so we can investigate a better approach later.